src/Symfony/Component/Console/Command/ListCommand.php

```diff
@@ -85,6 +86,7 @@ private function createDefinition()
             new InputArgument('namespace', InputArgument::OPTIONAL, 'The namespace name'),
             new InputOption('raw', null, InputOption::VALUE_NONE, 'To output raw command list'),
             new InputOption('format', null, InputOption::VALUE_REQUIRED, 'The output format (txt, xml, json, or md)', 'txt'),
+            new InputOption('show-hidden', null, InputOption::VALUE_NONE, 'To show hidden commands'),
```

Comment:
No opinion (as we can also use application descriptors in code, so as calling commands).